sources/vcdMaker/test/functional/test_001.std

```diff
@@ -0,0 +1,5 @@
+
+Parsed /home/wojtek/devel/vcdMaker/sources/vcdMaker/test/functional/test_001.txt: 
```

Comment:
You may want to consider removing sensitive information from path.

Reply:
Correct, the path will be shortened to just the file name.